docs/src/static-pages/accessibility.md

```diff
@@ -0,0 +1,478 @@
+---
+title: Developer Accessibility Guidelines
+subtitle: To ensure that we are building the best possible products that work for everyone, we need to consider accessibility at every stage.
+category: guidelines
+---
+
+# Accessible mixins for Web
+
+Within Backpack there is a number of SASS mixins available for use to enable content to be more accessible.
```

Comment:
```suggestion
Within Backpack there are a number of SASS mixins available for use to enable content to be more accessible.
```

Reply:
Good catch!

...Wait a second is @shaundon posing as you 😆

Reply:
I guess the student has become the master 😂